driver/pace/driver/run.py

```diff
@@ -116,82 +130,179 @@ class SerialboxConfig(InitializationConfig):
     """
 
     path: str
-    npx: int
-    npy: int
-    npz: int
-    layout: List[int]
+    serialized_grid: bool
     backend: str
+    rebuild: bool
+    validate_args: bool
+    format_source: bool
+    device_sync: bool
```

Comment:
Yes this is because we need to make a `stencil_config` to use TranslateUpdateDWindsPhys. When we include these grid variables in the gridinfo savepoint, it should look a bit cleaner.